### ISSUE TITLE
Configure SBS notebook still has hardcoded indices in microplot

### ISSUE THREAD

**Issue by mat10d** [cheeseman-lab/brieflow#21]
The microplots generated here:

```
g_aligned_microshow = microplot.Microimage(
    aligned[0, 1, :, :], cmaps="pure_green", channel_names=BASES[0]
)
t_aligned_microshow = microplot.Microimage(
    aligned[0, 2, :, :], cmaps="pure_yellow", channel_names=BASES[1]
)
a_aligned_microshow = microplot.Microimage(
    aligned[0, 3, :, :], cmaps="pure_magenta", channel_names=BASES[2]
)
c_aligned_microshow = microplot.Microimage(
    aligned[0, 4, :, :], cmaps="pure_cyan", channel_names=BASES[3]
)
```

Assume that bases are in indices 1,2,3,4. This needs to be fixed to take in the position corresponding to it in the channel names

```
CHANNEL_NAMES = ["DAPI", "G", "T", "A", "C"]
```

Low priority.

**Comment by mat10d** [comment]
Another issue I noticed was that annotate_segment_on_sequencing_data is pretty unreliable. The bug I am noticing is when you have more complex data (like 5 channels). Two changes:
1.  I think what we should be doing is just passing the cellpose_rgb into this. After all that is what is being segmented
2. I think we should set a default color for the outlines (and maybe not make them dotted

**Comment by roshankern** [comment]
Both resolved in #27
